conftest.py

```diff
@@ -448,10 +449,10 @@ def _skip_ported_msg(current_running_version, ported_from_version):
 @pytest.fixture(autouse=True)
 def fixture_ported_to_in_jvm(request, fixture_dtest_setup):
     """
-    Adds a new mark called 'ported_to_in_jvm' which denotes that a test was ported to jvm-dtest.
+    Adds a new mark called 'ported_to_in_jvm' which denotes that a test was ported to an in-JVM dtest.
 
-    As of this point in time there are weaknesses of jvm-dtest which require these tests to still
-    be run in the cases not covered by jvm-dtest; namely vnode.
+    In-JVM dtests do not currently support running with compression or vnodes, so tests that use this annotation will
```

Comment:
It seems that Python dtests aren't run with different compression options anymore, so we can remove the mention of compression, especially since the fixture code doesn't do anything about it.

Reply:
I think that JVM dtests are also missing the Python option `--use-off-heap-memtables`. Perhaps we should add it to this fixture so ported dtests are still run if that option is set, although we could do that in a separate ticket.

Reply:
I'm fine with the separate ticket option. In the meantime, I'll rebase and remove the mention of compression.